lib/Invoice.js

```diff
@@ -36,6 +36,8 @@ export class Invoice {
     this._options.logoImage = options.logoImage
     this._options.adjustmentInvoiceNumber = options.adjustmentInvoiceNumber
     this._options.prepaymentInvoice = options.prepaymentInvoice || false
+    this._options.finalInvoice = options.finalInvoice || false
```

Comment:
Please don't use the `false` default value. If we don't set `finalInvoice` then it will be not in the generated xml so we save some bandwidth and we don't break the test of the library users.